packages/apalis-cron/src/lib.rs

```diff
@@ -69,6 +69,9 @@
 //! }
 //! ```
 
+#[cfg(feature = "time")]
```

Comment:
Is there a point of adding a time feature in apalis-cron?

Reply:
I'm not sure :)
My main reasoning was to inform anyone enabling both `cron` and `time` features of their incompatibility as earlier as possible.